cms/locale/locale.py

```diff
@@ -127,9 +128,15 @@ def wrap_translations_for_tornado(trans):
     def translate(message, plural_message=None, count=None):
         if plural_message is not None:
             assert count is not None
-            return trans.ungettext(message, plural_message, count)
+            if six.PY3:
+                return trans.ngettext(message, plural_message, count)
+            else:
+                return trans.ungettext(message, plural_message, count)
         else:
-            return trans.ugettext(message)
+            if six.PY3:
+                return trans.gettext(message)
+            else:
+                return trans.ugettext(message)
```

Comment:
Optional: mild preference for assigning only once at the top `_gettext` and `_ngettext` to the correct variant